.storybook/config.js

```diff
@@ -1,4 +1,4 @@
-import { configure } from '@kadira/storybook'
+import { configure } from '@storybook/react'
```

Comment:
Not sure what the namespace "story" was, but everything's on the `@storybook` namespace now.